### ISSUE TITLE
Adjust contrast for handheld games (displaying non-activated elements)

### ISSUE THREAD

**Issue by famiclone6502** [mamedev/mame#7036]
As an enhancement, I think it would be helpful to allow the user to display the elements that aren't actively "lit" and adjust the contrast of how well you can see these elements. Even on "Game & Watch Gallery 4" on GBA, you could also see a similar implementation with the official simulations of their classic handheld games. 

When playing the physical versions of LCD, LED, VFD, or other similar handheld game screen technology, usually you can see the non-activated elements (depending on lighting, the angle of the screen, the strength of the battery, etc). 

For some games that I own and can directly compare, I've found that it is much more difficult to play without the context of these elements being visible. As an experiment, I created and displayed a background of all objects, and my score went up significantly and I was playing as well as I did on the original. 

It is debatable whether or not this is cheating, or if the designers intended these elements to be seen. But, in practice, it was usually visible even if it was faint. And again, even the official Game & Watch collection for GBA does this in classic mode.

**Comment by happppp** [comment]
On VFD games, yes the unlit segments are visible on the actual handhelds.
On Game & Watch, maybe some of them? Very very faintly. On Game & Watch Gallery remakes, Nintendo exaggerated it.
Here 2 random picks from youtube, I don't see the unlit elements.

https://www.youtube.com/watch?v=Y-Fiu7K18jo
https://www.youtube.com/watch?v=JCW9rQlmbxE

**Comment by famiclone6502** [comment]
Here is one example I found from YouTube of a G&W game showing all the segments: https://www.youtube.com/watch?v=ZhmaaEkoHEg

Most people do not record it at an angle, but it is usually visible in real life. Again it also depends on the lighting, battery strength, and other factors. 

One of the games I was having the most trouble with has a VFD display, as I couldn't always determine the path an object was traveling.

**Comment by happppp** [comment]
Right, here's another Nelsonic Game Watch, different video angle, different game, different person recording it. You can see the unlit segments. https://www.youtube.com/watch?v=vC0Srpc7L5Y

But, these are not official Nintendo G&W. They have a lesser quality LCD I suppose? Do Nintendo's ones have the effect prominently like that and am I just unlucky that every youtube video I click doesn't show it?

**Comment by DavidHaywood** [comment]
The segments are completely invisible when not lit on every official G&W unit I've used.

Only the very low quality / cheap ones seem to have that issue (one of the Radica Poker units does it)

Maybe it comes down to the factory where they were made, or there's an ageing factor?

**Comment by famiclone6502** [comment]
They are still official, just re-issues by another manufacturer. But I don't think it should be limited only to games that had versions released with lower quality screens. Many of the official Game & Watch Gallery remakes with this feature this did not have re-releases in the Mini Classics series, so there must have been a reason to display all the segments. 
 
The current video options for LCD games allow toggling the background, shadow, etc. Not all of these displays are necessarily realistic, but they are welcome user preferences. It could be implemented there as an on/off toggle for faintly showing all segments or even adjustable in the Slider Options for changing the contrast.
